src/language-client-mutualization.ts

```diff
@@ -237,12 +237,16 @@ function bindClientToServer (
               Object.entries(params.edit.changes).filter(([uri]) => isDocumentOpen(uri))
             )
             : undefined,
-          documentChanges: params.edit.documentChanges?.filter(documentEdit => {
-            if (TextDocumentEdit.is(documentEdit)) {
-              return isDocumentOpen(documentEdit.textDocument.uri)
-            }
-            return false
-          })
+          documentChanges: params.edit.documentChanges
+            ?.filter(TextDocumentEdit.is)
+            .filter(({ textDocument }) => isDocumentOpen(textDocument.uri) && documents.get(textDocument.uri) != null)
```

Comment:
Why checking `documents.get(textDocument.uri) != null`? isn't already what isDocumentOpen is doing?

Reply:
You're right, I didn't check !
[fix: remove unnecessary null check](https://github.com/CodinGame/languageserver-mutualized/pull/16/commits/5eda97f196731fa770256f739a00a0fe97b0d495)